### DISCUSSION TITLE
Feature Request: Option to Expand Text if ToC is Disabled

### DISCUSSION THREAD

**Discussion by ghost** [imfing/hextra#38] [Ideas]
**Feature Request** - Configuration option to allow text to span to the right if `toc: false` in the front matter.

**Comment by imfing** [comment]
For now, a viable approach is to create your own custom layout, e.g. `layouts/_default/custom.html`, based on the [`layouts/docs/single.html`](https://github.com/imfing/hextra/blob/main/layouts/docs/single.html) from the theme. 
So you could remove the line with the `toc` component completely.

https://github.com/imfing/hextra/blob/6d00cb32b0077ed12be520229658975b3fc0e100/layouts/docs/single.html#L4

Then, in the top part of your page's Markdown file, add `layout: custom` like this:
```yaml
title: your page
layout: custom
```

Note: the reason behind putting your custom layout under `layouts/_default` is because Hugo's template lookup order:

<img width="500" alt="image" src="https://github.com/imfing/hextra/assets/5097752/2eca554f-575a-4038-9abc-65bebbd161a1">
